src/server.c

```diff
@@ -7136,13 +7136,24 @@ void daemonize(void) {
 }
 
 void version(void) {
-    printf("Redis server v=%s sha=%s:%d malloc=%s bits=%d build=%llx\n",
-        REDIS_VERSION,
-        redisGitSHA1(),
-        atoi(redisGitDirty()) > 0,
-        ZMALLOC_LIB,
-        sizeof(long) == 4 ? 32 : 64,
-        (unsigned long long) redisBuildId());
+    if (server.sentinel_mode) {
```

Comment:
What do you think about this? 
```suggestion
    printf("Redis %s v=%s sha=%s:%d malloc=%s bits=%d build=%llx\n",
        server.sentinel_mode ? "sentinel" : "server",
        REDIS_VERSION,
        redisGitSHA1(),
        atoi(redisGitDirty()) > 0,
        ZMALLOC_LIB,
        sizeof(long) == 4 ? 32 : 64,
        (unsigned long long) redisBuildId());
```

Reply:
Both are fine, this one is more concise.